### PR TITLE
Add solvePseudoSteadyStateProblem() in cython interface

### PR DESCRIPTION
**Pull request by lavdwall**
This fixes my post in the Cantera Users' Group:
https://groups.google.com/d/msg/cantera-users/cZZXmLGPXaM/qpwBDh47GAAJ

### CODE REVIEW

File: interfaces/cython/cantera/kinetics.pyx [hunk 1]
```diff
@@ -366,6 +366,12 @@ cdef class InterfaceKinetics(Kinetics):
         coverages for a specified amount of time.
         """
         (<CxxInterfaceKinetics*>self.kinetics).advanceCoverages(dt)
+      
+    def advance_coverages_to_steady_state(self):
+        """
+        This method advances the surface coverages to (pseudo) steady state.
```
Comment (speth) [inline interfaces/cython/cantera/kinetics.pyx]: I know this comes from the name of the underlying method, but can you either explain what "pseudo" means here, or remove it from the description, depending on what you think makes more sense?

**Review by speth — CHANGES_REQUESTED** [review]
The implementation looks good to me. I just have a small question about the docstring.

File: interfaces/cython/cantera/kinetics.pyx (hunk 1 above)
Comment (lavdwall) [reply interfaces/cython/cantera/kinetics.pyx]: For me it is OK just to remove it from the description.

Comment (speth) [reply interfaces/cython/cantera/kinetics.pyx]: Can you push a revised version of the PR, then?

**Pull request merged**